src/tests/suite/daos_obj_ec.c

```diff
@@ -472,6 +472,169 @@ ec_partial_update_agg(void **state)
 	free(verify_data);
 }
 
+/* Submit a set of EC writes(defined by ec_ext), these writes should eventually
+ * compose one or N stripes, then wait for aggregation and verify these writes
+ * have been aggregated (no more replicated data in parity shard), also check
+ * if aggregated data are still correct.
+ */
+static void
+overlapped_update_agg(test_arg_t *arg, struct ec_ext *exts, int ext_nr)
+{
+	struct ioreq	req;
+	daos_recx_t	recx;
+	daos_obj_id_t	oid;
+	struct ec_ext	range = {0};
+	daos_off_t	data_off;
+	int		stripe_nr;
+	int		stripe_sz;
+	int		i;
+	char		*data;
+
+	if (!test_runable(arg, 6))
+		return;
+
+	daos_pool_set_prop(arg->pool.pool_uuid, "reclaim", "time");
```

Comment:
disabled here?  then enable it "time" after insert finish?  Probably does not matter.

Though default pool "reclaim" is "time", i.e. this is probably not needed?